Index: codelite-1.0.2674+dfsg/configure
===================================================================
--- codelite-1.0.2674+dfsg.orig/configure	2009-01-30 04:13:42.000000000 +0800
+++ codelite-1.0.2674+dfsg/configure	2009-01-30 04:16:10.000000000 +0800
@@ -21,7 +21,7 @@
 	exe_name="Runtime/codelite"
 	exe_name1="codelite"
 	is_plugin=$1
-	if [ "$arch" = "x86_64" ]; then
+	if [ "$arch" = "x86_64" ] || [ "$arch" = "ia64" ]; then
 		compiler_name=${compiler_name}" -fPIC -DON_64_BIT "
 		c_compiler_name=${c_compiler_name}" -fPIC -DON_64_BIT "
 	fi
