Index: codelite-1.0.2432/configure
===================================================================
--- codelite-1.0.2432.orig/configure	2008-11-15 04:06:43.000000000 +0800
+++ codelite-1.0.2432/configure	2008-11-18 15:28:28.000000000 +0800
@@ -371,7 +371,7 @@
 	echo "	@mkdir -p \$(DESTDIR)$prefix/bin" >> Makefile
 	echo "	@cp \$(EXE_NAME) \$(DESTDIR)$prefix/bin/" >> Makefile
 	if [ "$make_symlink" = "yes" ] ; then
-	  echo "	@ln -s \$(DESTDIR)$prefix/bin/\$(EXE_NAME_NO_PATH) \$(DESTDIR)$prefix/bin/CodeLite" >> Makefile
+	  echo "	@ln -s \$(EXE_NAME_NO_PATH) \$(DESTDIR)$prefix/bin/CodeLite" >> Makefile
 	  echo "	@chmod +x \$(DESTDIR)$prefix/bin/CodeLite" >> Makefile
 	fi
 	echo "	@cp Runtime/le_dos2unix.sh \$(DESTDIR)$prefix/bin/" >> Makefile
