Index: codelite-1.0.2759+dfsg/Runtime/codelite.desktop
===================================================================
--- codelite-1.0.2759+dfsg.orig/Runtime/codelite.desktop	2009-02-10 00:56:58.000000000 +0800
+++ codelite-1.0.2759+dfsg/Runtime/codelite.desktop	2009-02-10 00:57:05.000000000 +0800
@@ -1,10 +1,10 @@
 [Desktop Entry]
-Version=0.9
+Version=1.0
-Name=codelite
+Name=CodeLite
 Comment=An Open Source IDE for C/C++
 TryExec=codelite
 Exec=codelite %f
-Icon=/usr/local/share/codelite/images/cubes.png
+Icon=/usr/share/codelite/images/cubes.png
 Terminal=false
 X-MultipleArgs=false
 Type=Application
