Index: codelite-1.0.2785+dfsg/Runtime/codelite.desktop.template
===================================================================
--- codelite-1.0.2785+dfsg.orig/Runtime/codelite.desktop.template	2009-02-24 22:58:14.000000000 +0800
+++ codelite-1.0.2785+dfsg/Runtime/codelite.desktop.template	2009-03-03 04:13:46.000000000 +0800
@@ -1,9 +1,8 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=codelite
+Name=CodeLite
 Exec=codelite %f
 Icon=%%PREFIX%%/share/codelite/images/cubes.png
 Terminal=false
 Type=Application
-Categories=Development
+Categories=Development;
 StartupNotify=true
