Index: codelite-1.0.2432/Runtime/codelite.desktop
===================================================================
--- codelite-1.0.2432.orig/Runtime/codelite.desktop	2008-03-31 05:50:39.000000000 +0800
+++ codelite-1.0.2432/Runtime/codelite.desktop	2008-11-18 15:28:32.000000000 +0800
@@ -1,10 +1,10 @@
 [Desktop Entry]
-Version=0.9
+Version=1.0
 Name=CodeLite
 Comment=An Open Source IDE for C/C++
 TryExec=CodeLite
 Exec=CodeLite %f
-Icon=/usr/local/share/codelite/images/cubes.png
+Icon=/usr/share/codelite/images/cubes.png
 Terminal=false
 X-MultipleArgs=false
 Type=Application
