Description: Turn off automatic new-version checking.
diff --git a/LiteEditor/editorsettingsmiscpanel.cpp b/LiteEditor/editorsettingsmiscpanel.cpp
index 23df8dc9e..8dd812c55 100644
--- a/LiteEditor/editorsettingsmiscpanel.cpp
+++ b/LiteEditor/editorsettingsmiscpanel.cpp
@@ -93,7 +93,7 @@ EditorSettingsMiscPanel::EditorSettingsMiscPanel(wxWindow* parent)
     m_fileEncoding->SetSelection(iCurrSelId);
 
     m_singleAppInstance->SetValue(clConfig::Get().Read(kConfigSingleInstance, false));
-    m_versionCheckOnStartup->SetValue(clConfig::Get().Read(kConfigCheckForNewVersion, true));
+    m_versionCheckOnStartup->SetValue(clConfig::Get().Read(kConfigCheckForNewVersion, false));
     m_maxItemsFindReplace->ChangeValue(::wxIntToString(clConfig::Get().Read(kConfigMaxItemsInFindReplaceDialog, 15)));
     m_spinCtrlMaxOpenTabs->ChangeValue(::wxIntToString(clConfig::Get().Read(kConfigMaxOpenedTabs, 15)));
     m_choice4->SetStringSelection(
diff --git a/LiteEditor/frame.cpp b/LiteEditor/frame.cpp
index 9e33922c1..27ae73179 100644
--- a/LiteEditor/frame.cpp
+++ b/LiteEditor/frame.cpp
@@ -2563,11 +2563,6 @@ void clMainFrame::OnTimer(wxTimerEvent& event)
     clLogMessage("Using " + wxStyledTextCtrl::GetLibraryVersionInfo().ToString());
     if(::clIsCygwinEnvironment()) { clLogMessage("Running under Cygwin environment"); }
 
-    if(clConfig::Get().Read(kConfigCheckForNewVersion, true)) {
-        m_webUpdate = new WebUpdateJob(this, false, clConfig::Get().Read("PromptForNewReleaseOnly", false));
-        m_webUpdate->Check();
-    }
-
     // enable/disable plugins toolbar functionality
     PluginManager::Get()->EnableToolbars();
 
