Description: Turn off automatic new-version checking.
--- a/LiteEditor/editorsettingsmiscpanel.cpp
+++ b/LiteEditor/editorsettingsmiscpanel.cpp
@@ -77,7 +77,7 @@ EditorSettingsMiscPanel::EditorSettingsMiscPanel(wxWindow* parent)
     m_fileEncoding->SetSelection(iCurrSelId);
 
     m_singleAppInstance->SetValue(clConfig::Get().Read(kConfigSingleInstance, false));
-    m_versionCheckOnStartup->SetValue(clConfig::Get().Read(kConfigCheckForNewVersion, true));
+    m_versionCheckOnStartup->SetValue(clConfig::Get().Read(kConfigCheckForNewVersion, false));
     m_maxItemsFindReplace->ChangeValue(::wxIntToString(clConfig::Get().Read(kConfigMaxItemsInFindReplaceDialog, 15)));
     m_spinCtrlMaxOpenTabs->ChangeValue(::wxIntToString(clConfig::Get().Read(kConfigMaxOpenedTabs, 15)));
     m_choice4->SetStringSelection(
--- a/LiteEditor/frame.cpp
+++ b/LiteEditor/frame.cpp
@@ -2641,11 +2641,6 @@ void clMainFrame::OnTimer(wxTimerEvent&
         clLogMessage("Running under Cygwin environment");
     }
 
-    if(clConfig::Get().Read(kConfigCheckForNewVersion, true)) {
-        m_webUpdate = new WebUpdateJob(this, false, clConfig::Get().Read("PromptForNewReleaseOnly", false));
-        m_webUpdate->Check();
-    }
-
     // enable/disable plugins toolbar functionality
     PluginManager::Get()->EnableToolbars();
 
