From ece0470a219b0dc84aafb9dc6454f5b84230a519 Mon Sep 17 00:00:00 2001
From: Eran Ifrah <eran@eran-debian.eran-debian>
Date: Sun, 31 Jan 2016 16:20:02 +0200
Subject: [PATCH] Reverted commit 448f5c which hard-wired a 64-bit arch into
 CMakeLists.txt!

---
 CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7680576..f5e9d22 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,12 +37,8 @@ project( "CodeLite" )
 
 ## Generate compile_commands.json file for code completion
 set( CMAKE_EXPORT_COMPILE_COMMANDS 1 )
-
-if( UNIX AND NOT APPLE)
-    set( CMAKE_LIBRARY_ARCHITECTURE x86_64-linux-gnu ) # add search path needed by (at least) libssh and gtk2 libraries
-endif()
-
 set( CL_PREFIX "/usr" )
+
 if (CMAKE_CURRENT_LIST_DIR) # since cmake 2.8.3
     set( CL_SRC_ROOT ${CMAKE_CURRENT_LIST_DIR})
 else()
-- 
2.7.0

